index.html

```diff
@@ -349,6 +341,43 @@
     </section>
 
     <section>
+      <h2>Extensions to the <a>DocumentOrShadowRoot</a> Mixin</h2>
+
+      <dl title='partial interface DocumentOrShadowRoot' class='idl'>
+         <dt>readonly attribute Element? pointerLockElement</dt>
+         <dd>
+           <dfn title="pointerLockElement"></dfn>
+
+	   <p>If the element, which is set as the target for mouse events while the pointer is
+	   locked, is a
+           <a href="https://dom.spec.whatwg.org/#concept-shadow-including-descendant">shadow-including descendant</a> of the
+           <a href="http://dom.spec.whatwg.org/#context-object">context object</a>,
+	   return the result of the following algorithm:
+	   <dl>
```

Comment:
@hayatoito pointed out in the code review (for Blink) that strictly speaking "retargeting" is originally designed for event retargeting and differs from what `activeElement` does in that the result should be in the same tree as the context object:

> activeElement must be the result of the retargeting algorithm ..., **if the result and the context object are in the same tree**.

So eventually we have to put an algorithm for `activeElement`, `pointerLockElement`, and `fullscreenElement` which slightly differs from retargeting somewhere (in dom spec).

Fixed CAPITAL LETTER VARIABLES, input/output to make it look more natural in this spec.